awx/ui/client/src/templates/workflows/workflow-chart/workflow-chart.directive.js

```diff
@@ -23,9 +23,16 @@ export default ['moment', '$timeout', '$window', '$filter', 'TemplatesStrings',
         restrict: 'E',
         link: function(scope, element) {
 
+            // Quickly render the start text so we see how wide it is and know how wide to make the start
+            // node element.
+            const startNodeText = $(`<span class="WorkflowChart-node" style="visibility:hidden;"><span class="WorkflowChart-startText">${TemplatesStrings.get('workflow_maker.START')}</span></span>`);
+            startNodeText.appendTo(document.body);
+            const startNodeTextWidth = startNodeText.width();
+            startNodeText.remove();
```

Comment:
I like this approach :+1: 

The virtualization lib we picked for output does something like this as well - do a trial render to get the dimensions of dynamic content and then adjust accordingly in a final render.